pygfx/linalgtest.py

```diff
@@ -0,0 +1,294 @@
+"""
```

Comment:
I guess this module doesn't belong in this PR

Reply:
Arg, I shouldn't put stuff like this in the repo dir :/